pkg/client/helper.go

```diff
@@ -245,8 +246,9 @@ func InClusterConfig() (*Config, error) {
 		Host:        "https://" + net.JoinHostPort(os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")),
 		Version:     "v1beta3",
 		BearerToken: string(token),
-		// TODO: package certs along with the token
-		Insecure: true,
+		TLSClientConfig: TLSClientConfig{
```

Comment:
Sure, that works for me, although I don't think it's true of our certs. A warning in the logs may still be a good idea, to clue in a human wondering why it's not working.